plugins/woocommerce/src/Blocks/BlockTypes/EmptyMiniCartContentsBlock.php

```diff
@@ -39,14 +39,17 @@ protected function render( $attributes, $content, $block ) {
 	 * @return string Rendered block type output.
 	 */
 	protected function render_experimental_empty_mini_cart_contents( $attributes, $content, $block ) {
+		$wrapper_attributes = get_block_wrapper_attributes(
+			array(
+				'data-wp-bind--aria-hidden' => '!state.cartIsEmpty',
+				'data-wp-bind--hidden'      => '!state.cartIsEmpty',
+				'data-wp-interactive'       => 'woocommerce/mini-cart',
+			)
+		);
+
 		ob_start();
 		?>
-		<div 
-			data-wp-bind--aria-hidden="!state.cartIsEmpty"
-			data-wp-bind--hidden="!state.cartIsEmpty" 
-			data-wp-interactive="woocommerce/mini-cart" 
-			class="wp-block-woocommerce-empty-mini-cart-contents-block"			
-		>
+		<div <?php echo $wrapper_attributes; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>>
```

Comment:
I'm using this `// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped` in this and the rest of the blocks because the linter complains even if `get_block_wrapper_attributes` is safe. I've seen this pattern in other blocks like the product filters. In other cases, it is not a problem because they use `sprintf`.